packages/suite/src/utils/suite/stake.ts

```diff
@@ -556,3 +556,29 @@ export const getDaysToAddToPoolInitial = (validatorsQueue?: ValidatorsQueue) =>
 
     return daysToWait <= 0 ? 1 : daysToWait;
 };
+
+export const getInstantStakeType = (
```

Comment:
rebase to develop and add test for this please 🙏

Reply:
https://github.com/trezor/trezor-suite/pull/13267/commits/956a4043aa467b08f04408fd8607a17871383d41